### ISSUE TITLE
Quantity input field

### ISSUE THREAD

**Issue by Tecfan** [Revadike/EnhancedBarter#26]
Since Enhanced Barter is for power users, could the quantity select box be swapped out for an input field with increment and decrement support? Having more than 10 copies of a tradable is a PITA, especially if you have over 50 copies.

![image](https://user-images.githubusercontent.com/956878/124289838-09e98700-db53-11eb-904d-496a9027e280.png)

Simple mockup. 

Should support both quantity and increment (plain number or number with a + or - in front of it).

**Comment by Tecfan** [comment]
Mockup is a little silly since +7 is already an option, but inputs like "72", "+35" are harder to do currently.

**Comment by Revadike** [comment]
Yeah, barter's implementation of it is awful. Ideally, we make it better on the site natively.
